doc/nrf/includes/nrf_cloud_rest_sample_requirements.txt

```diff
@@ -0,0 +1,39 @@
+.. requirement_keysign_start
+
+This sample uses the `nRF Cloud REST API`_, which requires that your device have valid signing keys installed and registerred with `nRF Cloud`_.
```

Comment:
I can't (easily) move it quite that far up (since the .. requirement_keysign_start is overwritten differently for each sample), but I can certainly place it before "To obtain and register a valid signing key..." and try to merge the two sections you mention